src/frontend/src/tests/lib/utils/tokens.utils.spec.ts

```diff
@@ -301,12 +304,101 @@ describe('sumTokensTotalUsdBalance', () => {
 			{ ...ETHEREUM_TOKEN }
 		];
 
-		const result = sumTokensUsdBalance(tokens);
+		const result = sumTokensUiUsdBalance(tokens);
 		expect(result).toEqual(50);
 	});
 
 	it('should correctly calculate USD total balance when tokens list is empty', () => {
-		const result = sumTokensUsdBalance([]);
+		const result = sumTokensUiUsdBalance([]);
+		expect(result).toEqual(0);
+	});
+});
+
+describe('sumMainnetTokensUsdBalance', () => {
```

Comment:
i think here is `calculateTokenUsdBalance`?

Reply:
yes, of course